modules/nf-core/vcftools/tests/main.nf.test

```diff
@@ -0,0 +1,146 @@
+nextflow_process {
+
+    name "Test Process VCFTOOLS"
+    script "../main.nf"
+    process "VCFTOOLS"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "vcftools"
+
+    test("sarscov2 - vcf") {
+
+        config "./base.config"
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test' ], // meta map
+                            file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/vcf/test.vcf', checkIfExists: true)
+                        ]
```

Comment:
```suggestion
                input[0] = [
                    [ id:'test' ], // meta map
                    file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/vcf/test.vcf', checkIfExists: true)
                ]
```

Not important really, just code style: Fot multi-line input, I would have opening/closing bracket in separate lines.